securedrop_salt/sd-base-template.sls

```diff
@@ -19,9 +19,16 @@ sd-base-template:
     - tags:
       - add:
         - sd-workstation
-        - sd-{{ sdvars.distribution }}
+        - sd-{{ apt_config.distribution }}
     - features:
       - enable:
         - service.paxctld
     - require:
       - qvm: dom0-install-debian-minimal-template
+
+# Store signing key in salt cache so that it can later be provisioned to templates in their apt-sources files.
+# Using cp.push or cp.cache_file don't work due to the minion-minion setup.
```

Comment:
Sorry, this is going over my head. Why can't the step in fpf-apt-repo.sls just read the original `apt_config['keyfile']`?

Reply:
The fpf-apt-repo.sls steps are run on the Debian VMs, but the bootstrap keyring files live in dom0. The debian vms have access to /srv/salt (as `salt://`) and to the cache dir but not to dom0 files, so somewhere in a state that's run on dom0 (I just picked this file for simplicity since then we only do it once, at initial provisioning time, and after that the key is managed via the keyring package), we cache the key, and then the VMs can read it.